utils/server.py

```diff
@@ -44,7 +31,7 @@ def create_new_veto_filepath(self):
         path = f'/map-veto/{str(uuid.uuid1())}'
         return path
 
-    async def _handler(self, request: web.Request) -> web.Response:
+    async def _handler(self, request: web.Request) -> web.Response or web.FileResponse:
```

Comment:
```suggestion
    async def _handler(self, request: web.Request) -> Union[web.Response, web.FileResponse]:
```
Needs `from typing import Union` import

Reply:
Needs import still